src/EPR.RegulatorService.Facade.API/Controllers/InviteRegulatorController.cs

```diff
@@ -127,7 +132,8 @@ public async Task<IActionResult> EnrolInvitedUser([FromBody] EnrolInvitedUserReq
             }
             catch (Exception e)
             {
-                _logger.LogError($"Error when creating the enrollment for id {request.UserId}");
```

Comment:
The standard way to use Microsoft logger in other project is as follows
_logger.LogError("Error when creating the enrolment for id {UserId}", request.UserId);